pkg/api/2018-09-30-preview/api/types.go

```diff
@@ -89,6 +89,14 @@ const (
 
 // NetworkProfile contains configuration for OpenShift networking.
 type NetworkProfile struct {
+	// VnetID is expected to be empty or match
```

Comment:
Could we line up VnetID and PeerVnetID so they can share the comment?

Reply:
Moved VnetID field below VnetCIDR, edited the comment to describe contents of both fields.